### PR TITLE
Declare version 1.8.0

### PR DESCRIPTION
**Pull request by jesseviola-hpe**
## Summary and Scope

- Reverted build images to Jenkins from github workflows
- Delcare version 1.8.0 in Changelog
- Fixed copy directory library to silently fail and continue
- Adding new entrypoint for argo work

## Issues and Related PRs

_List and characterize relationship to Jira/Github issues and other pull requests. Be sure to list dependencies._

* Resolves [CASMINST-5816](https://jira-pro.its.hpecorp.net:8443/browse/CASMINST-5816)

## Pull Request Checklist

- [x] Target branch correct
- [x] Testing is appropriate and complete, if applicable
- [x] Is a new version being released? Update https://github.com/Cray-HPE/cf-gitea-import/wiki/CSM-Compatibility-Matrix
- [x] [HPC Product Announcement](https://cray.slack.com/archives/C026TVCSXLH) prepared, if applicable

### CODE REVIEW

**Review by ekamin-hpe — DISMISSED** [review]
LGTM. Not sure if my approval will count towards the requirement, but approving anyway.

File: Jenkinsfile.github [hunk 1]
```diff
@@ -0,0 +1,94 @@
+/*
+ *
+ *  MIT License
+ *
+ *  (C) Copyright 2021-2022 Hewlett Packard Enterprise Development LP
```
Comment (mharding-hpe) [inline Jenkinsfile.github:5]:
```suggestion
 *  (C) Copyright 2021-2023 Hewlett Packard Enterprise Development LP
```

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,40 @@
+#
+# MIT License
+#
+# (C) Copyright 2021-2022 Hewlett Packard Enterprise Development LP
```
Comment (mharding-hpe) [inline Makefile:4]:
```suggestion
# (C) Copyright 2021-2023 Hewlett Packard Enterprise Development LP
```

File: argo_entrypoint.sh [hunk 3]
```diff
@@ -0,0 +1,41 @@
+#!/bin/sh
+#
+# MIT License
+#
+# (C) Copyright 2020-2022 Hewlett Packard Enterprise Development LP
```
Comment (mharding-hpe) [inline argo_entrypoint.sh:5]:
```suggestion
# (C) Copyright 2020-2023 Hewlett Packard Enterprise Development LP
```

File: test/test_force_existing_branch.sh [hunk 4]
```diff
@@ -1,3 +1,29 @@
+#!/usr/bin/env bash
+#
+# MIT License
+#
+# (C) Copyright 2021-2022 Hewlett Packard Enterprise Development LP
```
Comment (mharding-hpe) [inline test/test_force_existing_branch.sh:5]:
```suggestion
# (C) Copyright 2021-2023 Hewlett Packard Enterprise Development LP
```

**Review by mharding-hpe — APPROVED** [review]
The copyright years need to be updated in the files. I added suggestions for the ones I could, but some didn't have edits close enough to allow me to do it

**Review by ekamin-hpe — DISMISSED** [review]

**Review by haasken-hpe — DISMISSED** [review]

**Review by mharding-hpe — DISMISSED** [review]

**Comment by mharding-hpe** [comment]
> LGTM. Not sure if my approval will count towards the requirement, but approving anyway.

Your approval counts double!

File: CHANGELOG.md [hunk 5]
```diff
@@ -5,6 +5,20 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [Unreleased]
```
Comment (jsollom-hpe) [inline CHANGELOG.md:8]: You can delete the [Unreleased] section on your release branch.

File: Jenkinsfile.github [hunk 6]
```diff
@@ -0,0 +1,94 @@
...
+        stage("Set Versions") {
+            steps {
+                // This function is defined in cms-meta-tools:vars/setVersionFiles.groovy
+                setVersionFiles()
+            }
+        }
+
+        stage("runBuildPrep") {
+            steps {
+                sh "make runbuildprep"
```
Comment (jsollom-hpe) [inline Jenkinsfile.github:62]: This is going to need the change to authenticate to artifactory if cms-meta-tools is going to succeed.

**Review by jsollom-hpe — CHANGES_REQUESTED** [review]
This needs changes for Artifactory authentication.

File: CHANGELOG.md (hunk 5 above)
Comment (jsollom-hpe) [reply CHANGELOG.md]: I removed it myself as github gave me the option to commit to your PR.

File: Jenkinsfile.github (hunk 6 above)
Comment (jsollom-hpe) [reply Jenkinsfile.github]: I went ahead and made the authentication change and committed it to your PR. I am not sure about the etiquette of that maneuver. I would have put it in as a suggested fix, but I didn't see that option when I went to edit the file. In this case, I didn't think you'd mind, but in general I'm not in the habit of committing additional changes to people's PRs without their consent.

**Review by jsollom-hpe — APPROVED** [review]
My approval is conditional on the rebuild succeeding. I triggered one this morning after making a change to do artifactory authentication.

**Review by rbak-hpe — APPROVED** [review]

**Pull request merged**